pkg/utils/configMapUtils.go

```diff
@@ -62,6 +62,41 @@ func ValidateConfigMaps(configMaps []v1alpha1.ConfigMap, engineDetails EngineDet
 	return validConfigMaps, errorList
 }
 
+// ValidateSecrets validates the secrets, before checking them.
+func ValidateSecrets(secrets []v1alpha1.Secret, engineDetails EngineDetails, clients ClientSets) ([]v1alpha1.Secret, []error) {
+	var errorList []error
+	var validSecrets []v1alpha1.Secret
+
+	for _, v := range secrets {
+		if v.Name == "" || v.MountPath == "" {
+			log.Infof("Unable to validate the Secret, with Name: %v , with mountPath: %v", v.Name, v.MountPath)
+			e := errors.New("Aborting Execution, Secret Name or mountPath is invalid")
+			errorList = append(errorList, e)
+			return nil, errorList
+		}
+
+		_, err := clients.KubeClient.CoreV1().Secrets(engineDetails.AppNamespace).Get(v.Name, metav1.GetOptions{})
+
+		if err != nil {
+			//errors = append(errors, err)
+			log.Infof("Unable to find ConfigMap with Name: %v", v.Name)
```

Comment:
:) I guess you mean secrets

Reply:
But why is it here in the first place? Not in secretUtils.go?

Reply:
done! The thing was some common utils as volumeBuilders, or volumeMounts, that was keeping them in same file